### DISCUSSION TITLE
revisit / reinterpretation of variants

### DISCUSSION THREAD

**Discussion by parlar** [Clinical-Genomics/scout#4912] [Ideas]
This is a general discussion point.

I recently had a meeting with Qiagen, where they mentioned that QCII includes functionality for automatically reinterpreting genetic variants, similar to Moon. Qiagen also noted that certain countries now have regulations requiring this capability, with Germany being one of them.

It's unclear to me how close Sweden is to adopting similar requirements, but it might be something we should keep in mind moving forward.

Have there been discussions on this in the Scout core team?

**Comment by dnil** [comment]
Hi, yes, important question, but one that is a bit outside the core Scout part. We do support two different rerunner/cold case thawing tools (https://github.com/Clinical-Genomics-Lund/scout-rerunner and https://github.com/Clinical-Genomics/melter), but this is currently on hold at least in Solna, with the general ethics intertepretation being that we are not quite allowed to do general screening in this way, and that the situation where a patient or treating clinician initiates the reanalysis is preferred, so that there is a preparation and desire to receive an answer. That said, we occasionally do reanalysis for new genes etc after an ethics review. Interesting to hear that this is changing in Europe! Question for SFMG / GMS etc perhaps?

**Comment by parlar** [comment]
Yes! Thanks, was not aware that you had a system for it. Sorry about posting off-topic.

**Comment by dnil** [comment]
No worries whatsoever - it's just that we can't do a lot about it right this moment. But happy to discuss in some form both criteria for "thawing" cases (eg new genes on gene panels, significant new bioinformatics tools/variant analysis modalities, ...) and the ethical challenges. Experience says the vast majority of patients/families are happy with returned results - but that experience is biased a bit by the situations we have done so were carefully chosen.

I noticed we have the melter repo locked - it is rather system specific - but maybe we could clean out anything sensitive and generalize a bit if there is interest in collaboration on it. It is mostly api based.

The Lund rerunner is open and also an integrated Scout extension.
